src/kompose-util.ts

```diff
@@ -15,7 +15,7 @@ import {
 } from './utilities'
 
 const komposeToolName = 'kompose'
-const stableKomposeVersion = 'v1.18.0'
+const stableKomposeVersion = 'v1.31.2'
```

Comment:
```suggestion
const stableKomposeVersion = 'v1.32.0'
```

Reply:
couldn't find the source branch for this PR to update this in there... please make sure we grab this last stable release if we're merging this soon 🙏🏼

Reply:
sounds good i can review as soon as we update this

Reply:
So I ended up opening a separate PR (https://github.com/Azure/k8s-bake/pull/93) since the original author archived their repo and I'm not sure they are following this thread. @davidgamero could you review it?

Reply:
yep sounds good